lib/jsdom/living/form-data.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+const formDataSymbols = require("./form-data-symbols");
+
+module.exports = function (core) {
+  const Blob = core.Blob;
+  const File = core.File;
+
+  class FormData {
```

Comment:
No, because Maps can only contain one value per key.
For example:
```javascript
var formData = new FormData();
formData.append("test1", "value1");
formData.append("test2", "value2");
formData.append("test1", "value3");
var request = new XMLHttpRequest();
request.open("POST", "/test");
request.send(formData);
```
Should send the payload below :
```
------WebKitFormBoundaryPTFAfBO65hSkn3gb
Content-Disposition: form-data; name="test1"

value1
------WebKitFormBoundaryPTFAfBO65hSkn3gb
Content-Disposition: form-data; name="test2"

value2
------WebKitFormBoundaryPTFAfBO65hSkn3gb
Content-Disposition: form-data; name="test1"

value3
------WebKitFormBoundaryPTFAfBO65hSkn3gb--
```
If I put the values in Map inside lists, the values will be grouped by key and that's not how FormData should work.